### PR TITLE
Remove NEW_USER_AIRDROP_AMOUNT (320-change-new-user-airdrop)

### PR DESCRIPTION
**Pull request by vakond**
Removed usage of environment variable NEW_USER_AIRDROP_AMOUNT.

### CODE REVIEW

File: proxy/common_neon/neon_instruction.py [hunk 1]
```diff
@@ -186,29 +186,10 @@ def createERC20TokenAccountTrx(self, token_info) -> Transaction:
...
-            decimals=9,
-            mint=ETH_TOKEN_MINT_ID,
-            program_id=TOKEN_PROGRAM_ID
-        ))
-        self.debug(f"Token transfer from token: {self.operator_neon_address}, owned by: {self.operator_account}, to token: "
-                    f"{associated_token_account}, owned by: {associated_token_account} , value: {NEW_USER_AIRDROP_AMOUNT}")
-        return transfer_instruction
-
-    def make_trx_with_create_and_airdrop(self, eth_account, code_acc=None) -> Transaction:
+    def make_trx_with_create(self, eth_account, code_acc=None) -> Transaction:
```
Comment (otselnik) [inline proxy/common_neon/neon_instruction.py]: I suggest removing the `make_trx_with_create` function entirely and using `make_create_eth_account_trx` directly.
`make_create_eth_account_trx` can now only return Transaction.

Comment (vakond) [reply proxy/common_neon/neon_instruction.py]: Very reasonable, thx.

File: proxy/testing/test_airdropping_with_no_faucet.py [hunk 2]
```diff
@@ -27,27 +30,31 @@ def setUpClass(cls) -> None:
...
         actual_balance_wei = self._web3.eth.get_balance(account.address, block_identifier=block_number)
         self.assertEqual(self._EXPECTED_BALANCE_WEI, actual_balance_wei)
 
     def test_airdrop_on_deploy(self):
         contract_owner: LocalAccount = self._web3.eth.account.create()
+        request_airdrop(contract_owner.address)
         contract = self._contract_deployer.compile_and_deploy_contract(contract_owner, self._CONTRACT_STORAGE_SOURCE)
         actual_balance_wei = self._get_balance_wei(contract.address)
-        self.assertEqual(self._EXPECTED_BALANCE_WEI, actual_balance_wei)
+        #====self.assertEqual(self._EXPECTED_BALANCE_WEI, actual_balance_wei)
```
Comment (vakond) [inline proxy/testing/test_airdropping_with_no_faucet.py]: Why is it expected for new contract to have non-zero balance?

Comment (rozhkovdmitrii) [reply proxy/testing/test_airdropping_with_no_faucet.py]: For testing purposes,
There are tests those assume tokens  on the contract address in open zeppelin set

Comment (rozhkovdmitrii) [reply proxy/testing/test_airdropping_with_no_faucet.py]: Be careful you can broke them

**Review by vasiliy-zaznobin — DISMISSED** [review]

**Comment by rozhkovdmitrii** [comment]
If we don't have airdrop without the faucet more then `proxy/testing/test_airdropping_with_no_faucet.py` should be dropped away

**Review by afalaleev — DISMISSED** [review]

File: proxy/testing/test_airdropping_with_no_faucet.py (hunk 2 above)
Comment (vakond) [reply proxy/testing/test_airdropping_with_no_faucet.py]: I've removed the file test_airdropping_with_no_faucet.py

**Review by rozhkovdmitrii — DISMISSED** [review]

**Review by otselnik — DISMISSED** [review]

File: proxy/docker-compose-test.yml [hunk 3]
```diff
@@ -61,10 +61,10 @@ services:
     image: neonlabsorg/proxy:${REVISION}
     environment:
       SOLANA_URL: http://solana:8899
+      FAUCET_URL: http://faucet:3333
```
Comment (s-medvedev) [inline proxy/docker-compose-test.yml:64]: Why do you need FAUCET_URL to know inside `proxy`-container?

Comment (vakond) [reply proxy/docker-compose-test.yml:64]: It is used in `.buildkite/steps/deploy-test.sh` to get and pass into container `uniswap-v2-core`. The uniswap container lives outside of containers from docker-compose.yml.

```
export FAUCET_URL=$(docker exec proxy bash -c 'echo "$FAUCET_URL"')
...
docker run --rm -ti --network=container:proxy \
     -e FAUCET_URL \
     --entrypoint ./deploy-test.sh \
     ${EXTRA_ARGS:-} \
     $UNISWAP_V2_CORE_IMAGE \
     all
```

File: proxy/testing/test_contract_reverting.py [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
-import eth_utils
 from web3 import exceptions as web3_exceptions
 from solana.rpc.api import Client as SolanaClient
 from eth_account.account import LocalAccount
 
-from .testing_helpers import SolidityContractDeployer
+from .testing_helpers import SolidityContractDeployer, request_airdrop
 from ..common_neon.emulator_interactor import decode_revert_message
 
+from solcx import install_solc
```
Comment (rozhkovdmitrii) [inline proxy/testing/test_contract_reverting.py]: Bring it into `proxy/testing/__init__.py`

Comment (vakond) [reply proxy/testing/test_contract_reverting.py]: Done

**Review by afalaleev — APPROVED** [review]

**Review by rozhkovdmitrii — APPROVED** [review]

File: proxy/run-test-proxy.sh [hunk 5]
```diff
@@ -31,7 +28,7 @@ for i in $(seq 1 $NUM_ACCOUNTS); do
     echo 'Create balance and mint token'
     TOKEN_ACCOUNT=$( (spl-token create-account --owner "$ID_FILE" "$NEON_TOKEN_MINT" || true) | grep -Po 'Creating account \K[^\n]*')
     echo "TOKEN_ACCOUNT=$TOKEN_ACCOUNT"
-    spl-token mint "$NEON_TOKEN_MINT" $(("$NEW_USER_AIRDROP_AMOUNT"*100000)) --owner /spl/bin/evm_loader-keypair.json -- "$TOKEN_ACCOUNT"
+    spl-token mint "$NEON_TOKEN_MINT" 10000000 --owner /spl/bin/evm_loader-keypair.json -- "$TOKEN_ACCOUNT"
```
Comment (ivandzen) [inline proxy/run-test-proxy.sh:31]: [minor]: Seems that this token balance is not needed anymore. It was used as budget for airdrops

File: proxy/plugin/solana_rest_api_tools.py [hunk 6]
```diff
@@ -59,15 +59,15 @@ def get_token_balance_gwei(client: SolanaClient, pda_account: str, *, logger) ->
...
 @logged_group("neon.Proxy")
-def get_token_balance_or_airdrop(client: SolanaClient, eth_account: EthereumAddress, *, logger) -> int:
+def get_token_balance_or_zero(client: SolanaClient, eth_account: EthereumAddress, *, logger) -> int:
     solana_account, nonce = ether2program(eth_account)
     logger.debug(f"Get balance for eth account: {eth_account} aka: {solana_account}")
 
     try:
         return get_token_balance_gwei(client, solana_account)
     except SolanaAccountNotFoundError:
         logger.debug(f"Account not found:  {eth_account} aka: {solana_account} - return airdrop amount")
```
Comment (ivandzen) [inline proxy/plugin/solana_rest_api_tools.py:69]: Please, edit log message according new behaviour

**Pull request merged**